### ISSUE TITLE
BLS threshold signatures?

### ISSUE THREAD

**Issue by angrymouse** [paulmillr/noble-curves#12]
It's said in readme that BLS can be used well for threshold signatures, with link to article that describes how it works cryptographically.
However I couldn't find how to do it using noble-curves. Is it possible currently?

**Comment by paulmillr** [comment]
See "Sign 3 msgs with 3 keys" example in readme. Threshold = aggregated sigs.

**Comment by angrymouse** [comment]
Yeah that's m-of-m signatures as I see... What about n-of-m ones? Is there possibility for actually "threshold"?

**Comment by paulmillr** [comment]
We don't provide out-of-box API for it, but could be simple to implement.

Resource for myself and others about m-of-n threshold sigs on bls: https://www.jcraige.com/threshold-bls-signatures

**Comment by angrymouse** [comment]
Would be happy to contribute in implementation of this if I would know cryptography 😂 
Good luck to you and noble... Seems really clean and understandible, especially after libsodium.
Will keep this issue opened because noble doesn't have m-of-n BLS signatures API support yet... Stay safe!
